inst/scripts/import_clinvar_data.R

```diff
@@ -0,0 +1,19 @@
+#lynx -accept_all_cookies=TRUE https://www.science.org/doi/suppl/10.1126/science.adg7492/suppl_file/science.adg7492_data_s1_to_s9.zip
```

Comment:
Taking another look, the journal is subscription based but it seems the AM authors have actually made their article and all supplemental materials "Free Access"! My feelings are that it should be fine for us to include. The `browseURL(...)` works great, thank you for the suggestion. I'll go ahead with this approach instead of `lynx`.